ddtrace/tracer/option_test.go

```diff
@@ -114,32 +114,20 @@ func TestTracerOptionsDefaults(t *testing.T) {
 		assert := assert.New(t)
 		tracer := newTracer(
 			WithSampler(NewRateSampler(0.5)),
-			WithServiceName("api-intake"),
 			WithAgentAddr("ddagent.consul.local:58126"),
 			WithGlobalTag("k", "v"),
 			WithDebugMode(true),
 			WithEnv("testEnv"),
 		)
 		c := tracer.config
 		assert.Equal(float64(0.5), c.sampler.(RateSampler).Rate())
-		assert.Equal("api-intake", c.serviceName)
```

Comment:
Why the changes to this test case?

Reply:
This specific test case was moved into the below `TestServiceName/with-service-name` case.
It just duplicates test coverage unnecessarily.

Reply:
Ah gotcha, okay!